tests/phpunit/tests/test-columns.php

```diff
@@ -18,11 +20,17 @@ public static function pllSetUpBeforeClass( PLL_UnitTest_Factory $factory ) {
 
 		$factory->language->create_many( 2 );
 
-		self::$editor = self::factory()->user->create( array( 'role' => 'editor' ) );
+		self::$editor        = self::factory()->user->create( array( 'role' => 'editor' ) );
+		self::$translator_fr = self::factory()->user->create( array( 'role' => 'editor' ) );
+		self::$translator_es = self::factory()->user->create( array( 'role' => 'editor' ) );
```

Comment:
```suggestion
		self::$editor        = self::factory()->user->create_and_get( array( 'role' => 'editor' ) );
		self::$translator_fr = self::factory()->user->create_and_get( array( 'role' => 'editor' ) );
		self::$translator_es = self::factory()->user->create_and_get( array( 'role' => 'editor' ) );
```
This way you can get rid of the `new User` bellow.